generator/internal/genclient/language/internal/rust/rust.go

```diff
@@ -101,7 +109,8 @@ func (c *Codec) FieldType(f *genclient.Field, state *genclient.APIState) string
 			return "Option<std::collections::HashMap<" + key + "," + val + ">>"
 		}
 		if strings.HasPrefix(m.ID, ".google.protobuf.") {
-			return "Option<String> /* TODO(#77) - handle " + f.TypezID + " */"
+			// TODO(#77): Better handling of well-known types
```

Comment:
BTW, this was a good idea.  My idea of putting TODOs in the generated code ... not so smart.